purchase_discount/report/purchase_report.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2017-2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+import odoo.addons.decimal_precision as dp
+
+
+class PurchaseReport(models.Model):
+    _inherit = "purchase.report"
+
+    discount = fields.Float(
+        string="Discount (%)", digits=dp.get_precision("Discount"), group_operator="avg"
```

Comment:
This gives a "Deprecated call to decimal_precision.get_precision" warning in Odoo

Reply:
Done ! Thanks for the feedback